templates/personal.php

```diff
@@ -1,57 +1,4 @@
 <?php
-/**
- * SPDX-FileCopyrightText: 2016-2024 Nextcloud GmbH and Nextcloud contributors
- * SPDX-FileCopyrightText: 2013-2016 ownCloud, Inc.
- * SPDX-License-Identifier: AGPL-3.0-or-later
- */
-script('richdocuments', 'richdocuments-personal');
+script('richdocuments', 'richdocuments-personal'); // This should load your personal.js bundle
```

Comment:
While we are modernizing `personal.php` a bit, it might be nice if we can change this to use the more modern way to add a script: 

```suggestion
\OCP\Util::addScript('richdocuments', 'richdocuments-personal'); // This should load your personal.js bundle
```

Reply:
Done